tools/rollup-es5/src/es5/Es5Tokens.ts

```diff
@@ -31,33 +31,67 @@ export const defaultEs5CheckTokens:IEs5CheckKeyword[] = [
         ]
     },
     {
-        funcNames: [ /Object\.(getOwnPropertySymbols)\(/g ],
-        errorMsg: "[%funcName%] is not supported in an IE/ES5 environment, use a helper function or add explicit check for existence",
+        funcNames: [ /\bObject\.(getOwnPropertySymbols)\(/g ],
+        errorMsg: "[%funcName%] is not supported in a IE/ES5 environment, use a helper function or add explicit check for existence",
         ignoreIds: [
             "tslib.es6",                    // tslib.es6 library has a pre existence check before usage
             "object-assign\\index.js",      // object-assign node module contains a pre existence check before usage
             "object-assign/index.js"        // object-assign node module contains a pre existence check before usage
         ]
     },
     {
-        funcNames: [ /([\w0-9]*)\.(trim)[\s]*\(/g ],
-        errorMsg: "[%funcName%] is not a supported string method in an IE/ES5 environment, use strTrim().",
+        funcNames: [ /([\w0-9]*)\.(trim|substr|trimEnd|trimStart|includes|padStart|padEnd)[\s]*\(/g ],
+        errorMsg: "[%funcName%] is not a supported string method in a IE/ES5 environment, use strTrim*(), strSubstr(), strIncludes().",
         ignoreFuncMatch: [
             "Util.trim",                            // Make sure this isn't a reference to Util.trim()
             "DataSanitizer.trim"                    // Make sure this isn't a reference to Util.trim()
         ]
     },
     {
         funcNames: [ /([\w0-9]*)\.(startsWith)[\s]*\(/g ],
-        errorMsg: "[%funcName%] is not a supported string method in an IE/ES5 environment, use strStartsWith().",
+        errorMsg: "[%funcName%] is not a supported string method in a IE/ES5 environment, use strStartsWith().",
         ignoreFuncMatch: [
         ]
     },
     {
         funcNames: [ /([\w0-9]*)\.(endsWith)[\s]*\(/g ],
-        errorMsg: "[%funcName%] is not a supported string method in an IE/ES5 environment, use strEndsWith().",
+        errorMsg: "[%funcName%] is not a supported string method in a IE/ES5 environment, use strEndsWith().",
         ignoreFuncMatch: [
         ]
+    },
+    {
+        funcNames: [ /([\w0-9]*)\.(find|findIndex|findLast|findLastIndex)[\s]*\(/g ],
+        errorMsg: "[%funcName%] is not supported array method in a IE/ES5 environment, use a helper function or add explicit check for existence.",
+        ignoreFuncMatch: [
+        ]
+    },
+    {
+        funcNames: [ /[^\w\"\']Date\.(now)\(/g ],
+        errorMsg: "[%funcName%] is not supported Date method in a IE/ES5 environment, use a helper function or add explicit check for existence"
+    },
+    {
```

Comment:
We didn't need most of these previously because TypeScript with ES3 targeting stopped us, now that we are targeting ES5 and we still support IE then we needed to add these new restrictions.